content/200-concepts/150-data-platform/08-data-proxy.mdx

```diff
@@ -63,28 +63,28 @@ datasource db {
+You can use the Data Proxy with Node.js (deployed in any environment where you can also deploy a normal project using Prisma Client), or with Edge runtimes like Cloudflare Workers and Vercel Edge Functions (Middleware).
 
 #### Node.js
 
-To use the Data Proxy with Node.js, set and define the environment variable as usual. See [Environment Variables](https://www.prisma.io/docs/guides/development-environment/environment-variables).
+To use the Data Proxy with Node.js, set and define the environment variable with your database connection string as usual. See [Environment Variables](https://www.prisma.io/docs/guides/development-environment/environment-variables).
 
-For guidance on how to configure environment variables in your deployment, please refer to your provider's documentation:
+For guidance on how to configure environment variables in your deployment, please refer to your provider's documentation. For example:
 
 - AWS: [Using AWS Lambda environment variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html)
 - Heroku: [Configuration and config vars](https://devcenter.heroku.com/articles/config-vars)
 
 #### Edge runtimes
 
-Edge runtimes, such as [Cloudflare Workers](https://developers.cloudflare.com/workers/) and [Vercel Edge Functions](https://vercel.com/docs/concepts/functions/edge-functions), cannot access files in the file system. This prevents Prisma Client from automatically loading `.env` files. Instead, you must use environment variables to pass the connection string to the Data Proxy. Depending on your provider, you might need to set the `DATASOURCE_URL` differently.
+Edge runtimes, such as [Cloudflare Workers](https://developers.cloudflare.com/workers/) and [Vercel Edge Functions](https://vercel.com/docs/concepts/functions/edge-functions), cannot access files in the file system. This prevents Prisma Client from automatically loading `.env` files. Instead, you must use environment variables to pass the connection string to the Data Proxy. Depending on your provider, you might need to set the `DATABASE_URL` differently.
 
 ##### Cloudflare Workers
 
 In Cloudflare Workers, you can define environment variables with either `wrangler` or the dashboard. For more information, see [Environment Variables](https://developers.cloudflare.com/workers/platform/environment-variables/) in the Cloudflare documentation.
 
 ##### Vercel Edge Functions
 
-In Vercel Edge Functions, you can define environment variables as you would in `Next.js`. For more information, see [Environment Variables](https://nextjs.org/docs/basic-features/environment-variables) in the Vercel Edge documentation.
+In Vercel Edge Functions, you can define environment variables as you would for Vercel Serverless Functions, e.g. in any `Next.js` project. For more information, see [Environment Variables](https://vercel.com/docs/concepts/projects/environment-variables) in the Vercel documentation.
```

Comment:
This section was not super clear what it was comparing to, and the link used was not actually the relevant one.